lib/bugsnag/integrations/mongo.rb

```diff
@@ -65,6 +65,23 @@ def leave_mongo_breadcrumb(event_name, event)
       Bugsnag.leave_breadcrumb(message, meta_data, Bugsnag::Breadcrumbs::PROCESS_BREADCRUMB_TYPE, :auto)
     end
 
+    ##
+    # Removes values from filter hashes, replacing them with '?'
+    #
+    # @param filter_hash [Hash] the filter hash for the mongo transaction
+    #
+    # @return [Hash] the filtered hash
+    def sanitize_filter(filter_hash)
+      filter_hash.map do |key, value|
+        if value && value.is_a?(Array)
+          filtered_or = value.map { |filter| sanitize_filter(filter) }
```

Comment:
This variable could be slightly misleading because an Array value doesn't necessarily mean it was an `$or` (it could have been `$and` or something else).